lib/FetchEnvs.js

```diff
@@ -29,14 +29,15 @@ const KIND_MAP = new Map([
 
 const LRU = require('lru-cache');
 const LruOptions = {
-  maxSize: 100000, // the max cache size
+  maxSize: parseInt(process.env.FETCHENVS_CACHE_SIZE) || 100000, // the max cache size
```

Comment:
remember to allow this to be set as in https://github.com/razee-io/MustacheTemplate/blob/master/kubernetes/MustacheTemplate/resource.yaml#L65-L70